client/client_test.go

```diff
@@ -9613,7 +9613,9 @@ func testExportAttestations(t *testing.T, sb integration.Sandbox, ociArtifact bo
 			if ociArtifact {
 				subject := att.Manifest.Subject
 				require.NotNil(t, subject)
-				require.Equal(t, bases[i].Desc, *subject)
+				require.Equal(t, bases[i].Desc.MediaType, subject.MediaType)
+				require.Equal(t, bases[i].Desc.Digest, subject.Digest)
+				require.Equal(t, bases[i].Desc.Size, subject.Size)
```

Comment:
Add a check in here that the platform is `nil` (and I guess `annotations` as well).